src/replay.ts

```diff
@@ -88,17 +88,21 @@ async function processMessages(messages: any[]) {
 }
 
 export async function run() {
-  // Check latest indexed MCI from db
-  const lastMci = await getLastMci();
-  console.log('[replay] Last MCI', lastMci);
+  while (true) {
+    try {
+      // Check latest indexed MCI from db
+      const lastMci = await getLastMci();
+      console.log('[replay] Last MCI', lastMci);
 
-  // Load next messages after latest indexed MCI
-  const messages = await getNextMessages(lastMci);
-  if (messages && messages.length > 0) {
-    await processMessages(messages);
```

Comment:
I think the reason for not adding try catch here is to catch if something is wrong with DB, else server may fail silently